lib/Server.js

```diff
@@ -724,7 +724,16 @@ class Server {
   }
 
   listen(port, hostname, fn) {
-    this.hostname = hostname;
+    if (hostname === 'local-ip') {
+      this.hostname =
+        internalIp.v4.sync() || '0.0.0.0' || internalIp.v6.sync() || '::';
```

Comment:
Isn't this a dead code, since '0.0.0.0' can be never `false`?

Reply:
Oh, yep, we need change it to `internalIp.v4.sync() || internalIp.v6.sync() || '0.0.0.0'`

Reply:
fixed here https://github.com/webpack/webpack-dev-server/pull/3128